### PR TITLE
Remove ember-parachute; Remove page QP

### PR DESCRIPTION
**Pull request by futa-ikeda**
-   Ticket: [No ticket]
-   Feature flag: n/a

## Purpose
- Remove ember-parachute
- Remove page query param from branded discover page

## Summary of Changes
- update package.json and yarn
- update branded discover controller and `OnSearchParams` type to exclude `page`

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]

**Comment by adlius** [comment]
Since when did we stop using ember-parachute?

**Comment by brianjgeiger** [comment]
@adlius https://github.com/CenterForOpenScience/ember-osf-web/pull/1980

File: app/register/route.ts [hunk 1]
```diff
@@ -18,4 +19,9 @@ export default class Register extends Route {
     model() {
         return this.store.createRecord('user-registration');
     }
+
+    setupController(controller: RegisterController, model: any, transition: Transition) {
+        super.setupController(controller, model, transition);
+        controller.setup();
+    }
```
Comment (brianjgeiger) [inline app/register/route.ts:26]: Is this necessary now? This seems like something that would just happen.

**Review by brianjgeiger — APPROVED** [review]

File: app/register/route.ts (hunk 1 above)
Comment (futa-ikeda) [reply app/register/route.ts:26]: So this is a little confusing to look at now as `setup` isn't an actual controller lifecycle hook (it was a [hook that ember-parachute added](https://github.com/offirgolan/ember-parachute#usage)). I tried using the controller's `init` hook to do some of that provider-fetching stuff, but the QPs weren't defined in at the `init` stage, so this was the next easiest solution I could think of. I could change the `setup` function name to something else to be a potentially less misleading?

Comment (brianjgeiger) [reply app/register/route.ts:26]: Ah, I see.

**Pull request merged**